.env.sample

```diff
@@ -6,6 +6,9 @@ VITE_NETWORK=hedera
 # For Hedera Testnet: https://testnet.hashio.io/api
 # For Hedera Mainnet: https://mainnet.hashio.io/api
 VITE_CHAIN_RPC_URL=https://testnet.hashio.io/api
+VITE_EXPLORER_URL=https://testnet.hashio.io/explorer
```

Comment:
Add more config for hedera network, these values will fill at src/config/hedera/common.ts